pkg/helpers/reader/reader.go

```diff
@@ -60,6 +61,12 @@ func (r *ResourceReader) Apply(fs embed.FS, config interface{}, files ...string)
 		Stream(bytes.NewReader(rawObjects), "local").
 		Flatten().
 		Do()
+	if r.dryRun {
+		// Ignore CRD not installed error when dry-run
+		rb.IgnoreErrors(func(err error) bool {
+			return err != nil && strings.Contains(err.Error(), "ensure CRDs are installed first")
```

Comment:
just curious, do we have a certain error type for this?

Reply:
seems no